client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-page-preview.tsx

```diff
@@ -26,6 +26,17 @@ interface PatternPagePreviewProps {
 const PATTERN_PAGE_PREVIEW_ITEM_VIEWPORT_HEIGHT = 500;
 const PATTERN_PAGE_PREVIEW_ITEM_VIEWPORT_WIDTH = 1080;
 
+// A copy of the title block in Creatio 2's page.html.
+const getPageTitlePattern = ( title: string ) => `
+	<div
+		class="wp-block-group has-global-padding is-layout-constrained wp-container-9 wp-block-group-is-layout-constrained"
+		style="margin-top:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--60)"
```

Comment:
Noob question (sorry): `--wp--preset--spacing--60` will be defined somewhere, right? 😄

Reply:
Yes! The pattern is rendered in an iframe, which will have these CSS variables defined (via [BlockRenderedContainer](https://github.com/Automattic/wp-calypso/blob/trunk/packages/block-renderer/src/components/block-renderer-container.tsx) IIRC)